### ISSUE TITLE
Global for $<map>-sprite-dimensions

### ISSUE THREAD

**Issue by phpmycoder** [Compass/compass#1702]
I find `$<map>-sprite-dimensions` very useful because it reduces the amount of typing I have to do on all of my sprited graphics (also very handy if I need to change a size by a few pixels). So I almost exclusively set this to true for all of my sprites.
I feel that this is common and believe that it would be beneficial to be able to override the default value of false by setting a global variable like `$sprite-dimensions` to true so that all sprites by default provide dimensions.

**Comment by vance** [comment]
Agree. is there some way to always use sprite dimensions? The use case for not using the sprite dimensions is, uh, about zero.
